vpr/src/pack/output_clustering.cpp

```diff
@@ -221,11 +219,8 @@ static void clustering_xml_open_block(pugi::xml_node parent_node, t_type_ptr typ
 							}
 						}
 						VTR_ASSERT(prev_edge < prev_pin->num_output_edges);
-						mode_of_edge =
-								prev_pin->output_edges[prev_edge]->interconnect->parent_mode_index;
-						VTR_ASSERT(
-								mode == nullptr || &pb_type->modes[mode_of_edge] == mode);
-						VTR_ASSERT(mode_of_edge == 0); /* for now, unused blocks must always default to use mode 0 */
```

Comment:
Was this assert removed intentionally?

Reply:
Fixed!

Sorry this was extracted from the pull request which removes this constraint.